controllers/message.js

```diff
@@ -13,9 +14,14 @@ const sendMessage = async (req, res) => {
 const getMessages = async (req, res) => {
   const { id: user } = req.user
   const { id: chat } = req.params
-  const { skip, limit } = req.query
+  const { skip, limit, message } = req.query
+  const searchCondition = { chat }
 
-  const messages = await messageService.getMessages({ user, chat }, parseInt(skip), parseInt(limit))
+  if (message) {
+    searchCondition.text = getRegex(message)
+  }
+
+  const messages = await messageService.getMessages({ user, chat }, parseInt(skip), parseInt(limit), searchCondition)
```

Comment:
you can pass only one object - match, for example, with user, chat and text properties

Reply:
fixed